frontend/src/app/header/header.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { TestBed, inject, async } from '@angular/core/testing';
+import { HttpModule } from '@angular/http';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { HeaderComponent } from './header.component';
+import { AuthService } from '../auth.service';
+
+
+describe('HeaderComponent', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        NgbModule.forRoot(),
+        HttpModule
+      ],
+      providers: [
+      ],
+      declarations: [
+        HeaderComponent
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the header', async(() => {
+    inject([AuthService], () => {
+      const fixture = TestBed.createComponent(HeaderComponent);
+      const header = fixture.debugElement.componentInstance;
+      expect(haeder).toBeTruthy();
+    });
+  ;});
```

Comment:
`);});` was missing at the end of file (properly formated) and You added only `;` and not at the end.

Just see how your parenthesis and brackets close.